mk/genlink-config.mk

```diff
@@ -24,11 +24,11 @@ endif
 LDSCRIPT	= generated.$(DEVICE).ld
 DEVICES_DATA = $(OPENCM3_DIR)/ld/devices.data
 
-genlink_family		:=$(shell $(OPENCM3_DIR)/scripts/genlink.py $(DEVICES_DATA) $(DEVICE) FAMILY)
-genlink_subfamily	:=$(shell $(OPENCM3_DIR)/scripts/genlink.py $(DEVICES_DATA) $(DEVICE) SUBFAMILY)
-genlink_cpu		:=$(shell $(OPENCM3_DIR)/scripts/genlink.py $(DEVICES_DATA) $(DEVICE) CPU)
-genlink_fpu		:=$(shell $(OPENCM3_DIR)/scripts/genlink.py $(DEVICES_DATA) $(DEVICE) FPU)
-genlink_cppflags	:=$(shell $(OPENCM3_DIR)/scripts/genlink.py $(DEVICES_DATA) $(DEVICE) CPPFLAGS)
+genlink_family		:=$(shell python "$(OPENCM3_DIR)/scripts/genlink.py" $(DEVICES_DATA) $(DEVICE) FAMILY)
+genlink_subfamily	:=$(shell python "$(OPENCM3_DIR)/scripts/genlink.py" $(DEVICES_DATA) $(DEVICE) SUBFAMILY)
+genlink_cpu		:=$(shell python "$(OPENCM3_DIR)/scripts/genlink.py" $(DEVICES_DATA) $(DEVICE) CPU)
+genlink_fpu		:=$(shell python "$(OPENCM3_DIR)/scripts/genlink.py" $(DEVICES_DATA) $(DEVICE) FPU)
+genlink_cppflags	:=$(shell python "$(OPENCM3_DIR)/scripts/genlink.py" $(DEVICES_DATA) $(DEVICE) CPPFLAGS)
```

Comment:
hrm, this is .... hairy.  I'll need more windows testing on this, and unfortunately it also has a lot of potential to break other systems, where "python" is no longer clear what it is, python2 or python3.    Are you building with msys-git? or inside cmd.exe itself?

Reply:
(the _quoting_ is unnecessary without the python call)

Reply:
I'm running it using atollic truestudio, which uses a built-in make.exe using cmd if i'm correct. This just fixed it for me. Thought I'd share my two cents. 

"it also has a lot of potential to break other systems". Definitely true.

Reply:
Did you try with _only_ the quoting?